src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/chatpageobject/ChatPage.java

```diff
@@ -131,29 +171,34 @@ public void verifyPrivateMessageIsHighlighted(String user) {
         "private message section is highlighted", true, driver);
   }
 
-  public void verifyPrivateChatTitle() {
-    wait.forElementVisible(privateChatHeader);
-    PageObjectLogging.log("verifyPrivateChatTitle", "private chat title is correct", true, driver);
+  public boolean isPrivateMessageHiglighted(String userName){
+    try {
+      return userWebElement(userName, PRIVATE_MESSAGE_SELECTED_USER_SELECTOR).isDisplayed();
+    }catch (ElementNotFoundException e){
+      return false;
+    }
+  }
+
+  public boolean isPrivateChatOpen() {
+    try {
+      return privateChatHeader.isDisplayed();
+    }catch (ElementNotFoundException e){
+      return false;
+    }
   }
 
   public void verifyMainChatIsHighlighted() {
     wait.forElementVisible(mainChatSelection);
     PageObjectLogging.log("verifyPrivateMessageIsHighlighted",
-        "private message section is highlighted", true);
+            "private message section is highlighted", true);
   }
 
-  public void verifyNormalUserDropdown(String userName) {
-    // This check is needed in case some of the previous tests failed
-    // We need to do the clean up in this place - allow provided user to send private messages
-    if (checkIfPrivateMessagesNotAllowed(userName)) {
-      allowPrivateMessageFromUser(userName);
+  public boolean isRegularUserDropdownDisplayed() {
+    if(userDropDownActionsElements.size() == REGULAR_USER_DROBDOWN_ELEMENTS){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this if-then-else statement by a single return statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1126)